src/main/java/com/epam/preprod/sirenko/command/GetProductsListCommand.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.preprod.sirenko.command;
+
+import com.epam.preprod.sirenko.DAO.ProductsInDBDAO;
+import com.epam.preprod.sirenko.entity.Product;
+
+import java.util.List;
+
+public class GetProductsListCommand extends Command{
+	
+	@Override
+	public List<Product> execute() {
+		ProductsInDBDAO productsInDBDAO = new ProductsInDBDAO();
```

Comment:
We should create all services, dao ets at program start, not during command execution

Reply:
couldn't do it